sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java

```diff
@@ -33,6 +33,7 @@
     private Long avgRt;
     private Long maxThread;
     private Double qps;
+    private Double avgCpu;
```

Comment:
Maybe it's better to be `maxCpuUsage`?

Reply:
I saw properties in [SystemRule ](https://github.com/alibaba/Sentinel/blob/master/sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRule.java) named highestSystemLoad and  it named avgLoad in [SystemRuleEntity](https://github.com/alibaba/Sentinel/blob/master/sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java) ,so i just do like to name highestCpuUsage in [SystemRule ](https://github.com/alibaba/Sentinel/blob/master/sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRule.java) to avgCpu in [SystemRuleEntity](https://github.com/alibaba/Sentinel/blob/master/sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java)

Reply:
Other legacy fields might not be good enough... For CPU usage it's an instantaneous value rather that average value.